scim-core/src/main/java/org/apache/directory/scim/core/json/ObjectMapperFactory.java

```diff
@@ -61,6 +61,7 @@ private static ObjectMapper createObjectMapper() {
     objectMapper.setAnnotationIntrospector(pair);
 
     objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    objectMapper.configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS, true);
```

Comment:
@bdemers @erant10 This was Eran's suggested fix when I asked him about handling "Add" vs "add" for PATCH operation values. Should probably also add a test for this somewhere, but I just wanted to get this on people's radar screen. Right now, we're getting failures during Azure AD SCIM syncs due to "Add" not being accepted as a valid PATCH operation value.

Reply:
Great find!
Given that one major vendor does this, I'm plus one.  It's not clear to me what the official RFC stance is, the closest I could find are these:

https://www.rfc-editor.org/rfc/rfc7644#section-3.10
https://datatracker.ietf.org/doc/html/rfc7643#section-2.1